src/main/java/org/mule/service/http/impl/service/client/async/ResponseBodyDeferringAsyncHandler.java

```diff
@@ -46,16 +53,28 @@
 public class ResponseBodyDeferringAsyncHandler implements AsyncHandler<Response> {
 
   private static final Logger logger = LoggerFactory.getLogger(ResponseBodyDeferringAsyncHandler.class);
+  private static final int SIZE_17_KB = 17 * 1024;
```

Comment:
add a comment explaining how you get to this number

Reply:
I think we should use the user's in this case and put the default at the attribute level.